### PR TITLE
feat: add participation to nodejs bindings

### LINKED ISSUE #1532: Implement `Account` Governance methods
## Description

:no_entry_sign: Blocked until technical specification is solidified.

The base functionality of registering Governance proposals for a Firefly profile will be necessary as part of a main user flow.

See `Account` [Methods](https://hackmd.io/@maxwellmattryan/r1zCOAaGs/%2FBH6LvsTYQWWjymIHQ061Pw#Methods1) in the technical specification.

## Motivation

Required for Firefly's Governance feature.

## Requirements

Refer to the methods in the [technical specification](https://hackmd.io/@maxwellmattryan/r1zCOAaGs/%2FBH6LvsTYQWWjymIHQ061Pw#Methods1).

## Open questions (optional)

N/a

## Are you planning to do it yourself in a pull request?

Some where I can.

### PR DESCRIPTION
**Pull request by Tuditi**
# Description of change

Make the participation methods of the account object callable through nodejs.

## Links to any relevant issues

closes: #1532 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Through example 28 by using a private tangle. The following curl adds an event to the private tangle:

```
curl -X POST http://localhost:14264/api/participation/v1/admin/events -H 'Content-Type: application/json' -d '{"name":"Tuditi Proposal","milestoneIndexCommence":100,"milestoneIndexStart":115,"milestoneIndexEnd":510700,"payload":{"type":0,"questions":[{"text":"Should Firefly release a token?","answers":[{"value":1,"text":"Yes","additionalInfo":""},{"value":2,"text":"No","additionalInfo":""}],"additionalInfo":""}]},"additionalInfo":"Nothing needed here"}'
```

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Can you add a .change please ?

File: bindings/nodejs/types/participation.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import type { OutputId } from './output'
+
+export interface ParticipationOverview {
+    participations: [EventId, [OutputId, ParticipationOverview]]
```
Comment (thibault-martinez) [inline bindings/nodejs/types/participation.ts]: TrackedParticipationOverview ?

File: bindings/nodejs/examples/29-add-voting-event.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/**
+ * This example creates a new voting event
+ */
+const getUnlockedManager = require('./account-manager');
+
+async function run() {
+    try {
+        const manager = await getUnlockedManager();
+        await manager.getP
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/29-add-voting-event.js]: Copy paste went wrong? :trollface:

Comment (Tuditi) [reply bindings/nodejs/examples/29-add-voting-event.js]: No, this is for another PR :trollface: , I removed it

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**